saleor/product/urls.py

```diff
@@ -10,4 +10,6 @@
         views.category_index, name='category'),
     url(r'(?P<slug>[a-z0-9-_]+?)-(?P<product_id>[0-9]+)/add/$',
         views.product_add_to_cart, name="add-to-cart"),
-]
+    url(r'^collection/(?P<pk>[0-9]+)/$',
```

Comment:
This URL could have similar format to the category URL (`slug` + `collection_id`).